metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/api/FakeBigtableConnection.java

```diff
@@ -185,7 +182,7 @@ public ColumnFamily createColumnFamily(
                         case SET_CELL:
                             final Mutation.SetCell setCell = mutation.getSetCell();
                             final ColumnFamily columnFamily =
-                                columnFamilies.get(Pair.of(table, setCell.getFamilyName()));
+                                table.getColumnFamily(setCell.getFamilyName()).get();
```

Comment:
use `Optional#orElseThrow()`, otherwise we get a non-descriptive error message when the optional is empty.